core/domain/classifier_services.py

```diff
@@ -305,9 +313,9 @@ def _update_classifier_training_job(classifier_training_job_model, status):
     classifier_training_job_model.put()
 
 
-def save_classifier_training_job(algorithm_id, exp_id, exp_version,
-                                 state_name, status, training_data,
-                                 job_id="None"):
+def save_classifier_training_job(algorithm_id, exp_id,
```

Comment:
@seanlip Again, a separate update job function won't really be required since it can only update the status. Nothing else is mutable. Fundamentally, mark_training_job_complete has to do the same thing. Access the datastore, change status to COMPLETE and save it back. We are doing pretty much the same thing here.